src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardElements.kt

```diff
@@ -435,14 +451,14 @@ private fun getLobbyDisplayPair(): List<ScoreboardElementType> {
 private fun getPowerDisplayPair() = listOf(
     when (MaxwellAPI.currentPower) {
```

Comment:
use `maxwel api . current power?.let{block} ?: "open your bags"` instead of `when`